website/docs/r/managed_devops_pool.html.markdown

```diff
@@ -0,0 +1,314 @@
+---
+subcategory: "Managed DevOps Pools"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_devops_pool"
+description: |-
+  Manages a Managed DevOps Pool.
+---
+
+# azurerm_managed_devops_pool
+
+Manages a Managed DevOps Pool.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_user_assigned_identity" "test" {
+  name                = "example-uai"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_dev_center" "test" {
+  name                = "example-devcenter"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+}
+
+resource "azurerm_dev_center_project" "example" {
+  dev_center_id       = azurerm_dev_center.example.id
+  location            = azurerm_resource_group.example.location
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_managed_devops_pool" "example" {
+  name                           = "example"
+  resource_group_name            = azurerm_resource_group.example.name
+  location                       = azurerm_resource_group.example.location
+  dev_center_project_resource_id = azurerm_dev_center_project.example.id
+  maximum_concurrency            = 1
+
+  azure_devops_organization_profile {
+    organization {
+      parallelism = 1
+      url         = "https://dev.azure.com/example"
+    }
+
+    permission_profile_kind = "SpecificAccounts"
+
+    administrator_accounts {
+      groups = ["group1@example.com", "group2@example.com"]
+      users  = ["user1@example.com", "user2@example.com"]
+    }
+  }
+
+  stateless_agent_profile {
+    automatic_resource_predictions_profile {
+      prediction_preference = "Balanced"
+    }
+  }
+
+  vmss_fabric_profile {
+    sku {
+      name = "Standard_D2ads_v5"
+    }
+
+    image {
+      resource_id = "/Subscriptions/00000000-0000-0000-0000-000000000000/Providers/Microsoft.Compute/Locations/australiaeast/publishers/canonical/artifacttypes/vmimage/offers/0001-com-ubuntu-server-focal/skus/20_04-lts-gen2/versions/latest"
+      buffer      = "*"
+    }
+
+    storage_profile {
+      os_disk_storage_account_type = "Standard"
+
+      data_disk {
+        disk_size_gb = 1
+        drive_letter = "B"
+      }
+    }
+
+    network_profile {
+      subnet_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `dev_center_project_resource_id` - (Required) The ID of the Dev Center project.
```

Comment:
Can we make sure the `name`, `resource_group_name` and `location` are positioned first, to be consistent with the other documents?